frontend/src/components/event-card/EventDetail.jsx

```diff
@@ -0,0 +1,85 @@
+import React, { Component } from 'react';
+import styled from 'styled-components';
+import Typography from '../typography';
+import Icon from 'antd/lib/icon';
+
+const ListStyled = styled.li`
+  list-style: none;
+  margin: 0.5rem 1rem;
+
+  i {
+    color: ${props => props.color};
+    margin-right: 0.5rem;
+    font-weight: 800;
+  }
+`;
+
+export const ListItem = props => (
```

Comment:
You could desctucture the props attribute here, to make it a bit more clear which props are needed in this ListItem-component. eg. `({style, color, iconName, title, content})`

Reply:
sure